client/client-lib/src/lib.rs

```diff
@@ -188,25 +191,39 @@ impl PaymentParameters {
     }
 }
 
+// TODO: `get_module` is parsing `serde_json::Value` every time, which is not best for performance
 impl<T: AsRef<ClientConfig> + Clone> Client<T> {
     pub fn ln_client(&self) -> LnClient {
         LnClient {
-            config: &self.config.as_ref().ln,
+            config: self
+                .config
+                .as_ref()
+                .get_module::<LightningModuleClientConfig>("ln")
```

Comment:
Can we maybe even avoid supplying the `"ln"` string here if the client config trait had an associated constant which is this `&'static str`?

Reply:
Yes, though almost everything here will have to change anyway.